dist/feature-packs/eap7.2/wildfly-feature-pack-build.xml

```diff
@@ -20,9 +20,9 @@
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<build xmlns="urn:wildfly:feature-pack-build:2.0">
+<build xmlns="urn:wildfly:feature-pack-build:3.0" producer="jboss-server-migration-eap@maven(org.jboss.universe:community-universe):current">
     <dependencies>
-        <dependency group-id="org.wildfly.core" artifact-id="wildfly-core-galleon-pack">
+        <dependency group-id="org.wildfly.core" artifact-id="wildfly-core-galleon-pack" producer="wildfly-core@maven(org.jboss.universe:community-universe):current">
```

Comment:
Just for the info, since Galleon 2.0.0.Beta3, the producer attribute for a dependency is no longer necessary and an invalid configuration. The org.wildfly.core:wildfly-core-galleon-pack version will be inferred from the maven pom declared dependency.

Reply:
@yersan Thanks for info. I'm going to close this one now, as I changed my automation and no longer use (maintain) this. But is can be used as a starting point for further works on this.